prefect_gcp/credentials.py

```diff
@@ -72,6 +72,9 @@ class GcpCredentials:
         project: Name of the project to use.
     """
 
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/4CD4wwbiIKPkZDt4U3TEuW/c112fe85653da054b6d5334ef662bec4/gcp.png?h=250"  # noqa
+    _block_type_name = "Google Cloud Platform Credentials"
+
     service_account_file: Optional[Union[str, Path]] = None
```

Comment:
I'd like to narrow this type because it simplifies display in the UI.
```suggestion
    service_account_file: Optional[Path] = None
```